core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -137,11 +145,8 @@ object RequestChannel extends Logging {
       }
     }
 
-    def requestDesc(details: Boolean): String = {
-      val forwardDescription = envelope.map { request =>
-        s"Forwarded request: ${request.context} "
-      }.getOrElse("")
-      s"$forwardDescription$header -- ${loggableRequest.toString(details)}"
+    def requestDesc: JsonNode = {
```

Comment:
It seems that this one is used from `KafkaApis` and others. It may be better to keep the original implementation for these cases.